packages/js/src/generators/setup-verdaccio/files/config.yml

```diff
@@ -26,4 +22,7 @@ packages:
 logs:
   type: stdout
   format: pretty
-  level: http
+  level: warn
```

Comment:
@xiongemi I could not get this test to work unless I brought this down to `warn`... Do you have any idea why?